corehq/apps/app_manager/templates/app_manager/v1/form_view_base.html

```diff
@@ -259,6 +259,10 @@
             $('#cloudcare-preview-url').click(function(e) {
                 ga_track_event('CloudCare', 'Click "Preview Form"');
                 analytics.workflow("Clicked Preview Form");
+                {% if request.couch_user.days_since_created == 0 %}
+                    ga_track_event('CloudCare', 'Click "Preview Form" within first 24 hours');
+                    analytics.workflow("Clicked Preview Form within first 24 hours");
```

Comment:
Not blocking, but I'm curious why we need to have these two very similar analytics calls—but I assume there's a good reason. Also, I imagine there's no reason the text on the two is slightly different?

- Click "Preview Form" within first 24 hours
- Clicked Preview Form within first 24 hours